components/raftstore/src/store/worker/raftlog_gc.rs

```diff
@@ -149,6 +135,20 @@ impl<EK: KvEngine, ER: RaftEngine, R: CasualRouter<EK>> Runner<EK, ER, R> {
                 }
             }
         }
+        let start = Instant::now();
+        match self.gc_raft_log(groups) {
```

Comment:
This reorder the `Task::Purge` and `Task::Gc`, I'm not sure if it's OK. /cc @tabokie

Reply:
It's fine, they are unrelated.